Sources/MongoSwift/MongoSwift.swift

```diff
@@ -1,39 +1,29 @@
 import mongoc
 
-/// A utility class for libmongoc initialization and cleanup.
-public final class MongoSwift {
-    /// The version of `MongoSwift`.
-    public static let versionString = "0.1.0"
+private final class MongocInitializer {
+    private static let versionString = "0.1.0"
+    internal static let shared = MongocInitializer()
 
-    private final class MongocInitializer {
-        static let shared = MongocInitializer()
-
-        init() {
-            mongoc_init()
-            mongoc_handshake_data_append("MongoSwift", versionString, nil)
-        }
+    private init() {
+        mongoc_init()
+        mongoc_handshake_data_append("MongoSwift", MongocInitializer.versionString, nil)
     }
+}
 
-    /**
-     * Initializes libmongoc.
-     *
-     * This function should be called once at the start of the application
-     * before interacting with the driver.
-     */
-    public static func initialize() {
-        _ = MongocInitializer.shared
-    }
+/// Initializes libmongoc. Repeated calls to this method have no effect.
+internal func initializeMongoc() {
+    _ = MongocInitializer.shared
+}
 
-    /**
-     * Cleans up libmongoc.
-     *
-     * This function should be called once at the end of the application. Users
-     * should not interact with the driver after calling this function.
-     */
-    public static func cleanup() {
-        /* Note: ideally, this would be called from MongocInitializer's deinit,
-         * but Swift does not currently handle deinitialization of singletons.
-         * See: https://bugs.swift.org/browse/SR-2500 */
-        mongoc_cleanup()
-    }
+/**
+ * Release all memory and other resources allocated by libmongoc.
+ *
+ * This function should be called once at the end of the application. Users
+ * should not interact with the driver after calling this function.
+ */
+public func cleanupMongoc() {
```

Comment:
open to any other name ideas

Reply:
Even though this function only exists because of the mongoc dependency, I personally think it would be better if we phrased it in terms of MongoSwift. 

i.e. the function would be `cleanupMongoSwift` and the docstring would say something like "Release all memory and other resources allocated by MongoSwift". Note we may want to say "internal memory and resources", since any objects the user holds on to will continue to hold memory even after cleanup is performed.

Reply:
renamed and updated docstring.